src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -413,14 +424,6 @@ describe("Editor Page Component", () => {
             canvas.clearRegions = clearRegions;
         });
 
-        function dispatchKeyEvent(key: string, eventType: KeyEventType = KeyEventType.KeyDown) {
-            window.dispatchEvent(new KeyboardEvent(
-                eventType, {
-                    key,
-                },
-            ));
-        }
-
```

Comment:
Why are you getting rid of this function?

Reply:
I think I moved it somewhere else... I'll look into it